.github/workflows/version_bump.yaml

```diff
@@ -0,0 +1,25 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "**/*.go"
+      - "**/*.c"
+      - "**/*.h"
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: '0'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.26.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        WITH_V: false
```

Comment:
is there any reason why WITH_V is false?

Reply:
i don't think there's a reason, but this is what i copied from my 3.x branch. do you have a reason to enable it? i have no preference at this point.